client/my-sites/site-settings/form-writing.jsx

```diff
@@ -140,20 +136,6 @@ const SiteSettingsFormWriting = React.createClass( {
 				</SectionHeader>
 				<Card className="site-settings">
 					<FormFieldset>
-						<QueryTerms siteId={ this.props.siteId } taxonomy="category" />
```

Comment:
Actually, I removed it because I noticed it was removed in @folletto's design https://cloud.githubusercontent.com/assets/4389/20131188/d95b5870-a696-11e6-85ee-68620d3c069b.png